examples/pylab_examples/contour_image.py

```diff
@@ -30,7 +30,7 @@
 norm = cm.colors.Normalize(vmax=abs(Z).max(), vmin=-abs(Z).max())
 cmap = cm.PRGn
 
-plt.figure()
+plt.figure(figsize=(10,12))
```

Comment:
The Travis failure is due to PEP8, There is a missing space after the , as in `figsize=(10, 12)`